config.yml

```diff
@@ -272,5 +252,5 @@ cartpole:
   ccrc_weight: 1.0
 
 
-data_generator:
+data_generator: # FIXME: Warum seed for data_generator here?
   seed: 1  # If null random seed based on datetime is used
```

Comment:
This can be removed. Then, only need to edit the `run_data_generator_legacy.py` and load the seed from `config_data_gen.yml`. (in case we want to retain compatibility)

Reply:
Implemented! I am not testing run_data_generator_legacy.py though. It should be fine now, otherwise it is of little relevance I guess... Normal data generator tested & working.